commands/version/index.js

```diff
@@ -556,6 +578,32 @@ class VersionCommand extends Command {
 
     return gitPush(this.gitRemote, this.currentBranch, this.execOpts);
   }
+
+  createGitHubReleases() {
+    this.logger.info("github", "Creating releases...");
+
+    const client = createGitHubClient();
+
+    return Promise.all(
+      this.releaseNotes.map(({ notes, name }) => {
+        const tag = name === "root" ? this.tags[0] : this.tags.find(t => t.startsWith(name));
+
+        if (!tag) {
+          return Promise.resolve();
+        }
+
+        return client.repos.createRelease({
+          owner: "", // TODO
+          repo: "", // TODO
```

Comment:
Could `owner` and `repo` be derived from project metadata? (root `package.json` `repository.url` field?)

Reply:
I ended up just calling `git` since that package field might not always exist.